src/pages/acknowledgements.html.js

```diff
@@ -19,14 +19,14 @@ const Acknowlegements = ({data, location}) => (
     <Container>
       <div css={sharedStyles.articleLayout.container}>
         <div css={sharedStyles.articleLayout.content}>
-          <Header>Acknowledgements</Header>
+          <Header>Podziękowania</Header>
           <TitleAndMetaTags
             ogUrl={`${urlRoot}/acknowledgements.html`}
-            title="React - Acknowledgements"
+            title="React - Podziękowania"
           />
 
           <div css={sharedStyles.markdown}>
-            <p>We'd like to thank all of our contributors:</p>
+            <p>Chcielibyśmy podziękować naszym kontrybutorom:</p>
```

Comment:
Nie mogę się pogodzić z _kontrybutorami_
```suggestion
            <p>Chcielibyśmy podziękować osobom, które wniosły wkład w powstanie Reacta:</p>
```
ew

```suggestion
            <p>Chcielibyśmy podziękować osobom, które wnoszą wkład w powstawanie Reacta:</p>
```